LandonHotel.Tests/BookingServiceTests.cs

```diff
@@ -71,11 +71,15 @@ public void IsBookingCostForAdditionalPersonsCorrect()
             //TODO: Calculate total booking cost using the constant values and the business rule
             //           cost per day = rate + rate * guestCount * 10/100
 
-            var totalBookingCost = 1.0; //dummy value to start
+            var totalBookingCost = (rate + (rate * (guestCount-1) * 10/100)) * numberDays; //first guest rate, and add rate for (guests excluding the first guest) at 10% increase, multiplied by number of days
 
             //Act
             //TODO: Setup roomRepo and create new Booking using the constants
```

Comment:
You are missing the roomRepo setup (see line 54 above), which is why your code in the BookingService is returning null since your test method calls the BookingService with a null roomRepo.